common/utils/include/claragenomics/utils/device_buffer.cuh

```diff
@@ -32,26 +31,24 @@ public:
 };
 
 template <typename T>
-class device_storage
+class device_buffer
 {
 public:
     using value_type = T;
```

Comment:
`value_type` is unused, can be removed.

Reply:
I added the `value_type` to ease writing generic functions. However, yes, the interface of the class is cleaner if I use `value_type` in the return types instead of `T`.